### PR TITLE
Fix OrbitGl/Timer::Elapsed{Millis,Seconds}

### PR DESCRIPTION
**Pull request by dpallotti**
Bug: http://b/172329387

### CODE REVIEW

File: OrbitGl/Timer.h [hunk 1]
```diff
@@ -19,12 +19,12 @@ class Timer {
 
   [[nodiscard]] uint64_t ElapsedNanos() const { return EndOrNow() - start_; }
   [[nodiscard]] double ElapsedMicros() const { return static_cast<double>(ElapsedNanos()) * 0.001; }
-  [[nodiscard]] double ElapsedMillis() const { return ElapsedMicros() * 0.0001; }
-  [[nodiscard]] double ElapsedSeconds() const { return ElapsedMillis() * 0.0001; }
+  [[nodiscard]] double ElapsedMillis() const { return ElapsedMicros() * 0.001; }
```
Comment (dimitry-) [inline OrbitGl/Timer.h:22]: maybe change it to "/ 1000.0"? It seems more readable that way.

**Review by dimitry- — APPROVED** [review]

File: OrbitGl/Timer.h (hunk 1 above)
Comment (dpallotti) [reply OrbitGl/Timer.h]: Sounds fair, done.

**Comment by pierricgimmig** [comment]
Oh wow, talk about a noob mistake! It'll teach me to not test a last my additional change to a PR, thanks for the fix @dpallotti

**Review by pierricgimmig — APPROVED** [review]

**Review by florian-kuebler — APPROVED** [review]
LGTM. Thanks

**Pull request merged**